src/helpers.ts

```diff
@@ -81,3 +82,19 @@ function setFrameDimensions(frame: HTMLIFrameElement, options: EmbedOptions = {}
     frame.setAttribute('style', 'width:100%;');
   }
 }
+
+function setFrameAllowList(
+  target: HTMLElement & { allow?: string },
+  frame: HTMLIFrameElement,
+  options: EmbedOptions = {}
+) {
+  const allowList = target.allow?.split(';')?.map((key) => key.trim()) ?? [];
```

Comment:
We copy the allow list which is already part of the iframe, and add `cross-origin-isolated` if it's not yet there.